packages/carbon-react/src/components/PaginationNav/PaginationNav.stories.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { PaginationNav } from 'carbon-components-react';
+
+const props = () => ({
+  loop: false,
+  page: 0,
+  totalItems: 10,
+  itemsShown: 10,
+});
+
+export default {
+  title: 'Components/PaginationNav',
+  decorators: [(story) => <div style={{ width: '800px' }}>{story()}</div>],
+
+  parameters: {
+    component: PaginationNav,
+  },
+};
+
+export const _PaginationNav = () => <PaginationNav {...props()} />;
+
+_PaginationNav.storyName = 'PaginationNav';
+
+_PaginationNav.parameters = {
+  info: {
+    text: `
+         Pagination Nav is a group of pagination buttons.
+           `,
+  },
+};
```

Comment:
I think we can just get rid of this stuff 👍🏻

Reply:
done!